src/main/java/org/skriptlang/skript/lang/experiment/ExperimentData.java

```diff
@@ -0,0 +1,153 @@
+package org.skriptlang.skript.lang.experiment;
+
+import ch.njol.skript.Skript;
+import ch.njol.util.StringUtils;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Container for holding {@link Experiment}s that must be enabled or disabled to use.
+ */
+public class ExperimentData {
```

Comment:
I might prefer to see a builder for this and the `ExperimentData` itself be final. Would like other opinions

Reply:
yeah no big thoughts either way a dedicated builder would be more proper but in this case it doesn't really matter much imo. Probably best to do it properly regardless